python/device/device.py

```diff
@@ -269,6 +297,32 @@ def _configure_azure_monitor_metrics(self):
             "patches with remove operations(s)",
         )
 
+        self.reporter.add_float_measurement(
+            MetricNames.LATENCY_QUEUE_MESSAGE_TO_SEND,
+            "Number of milliseconds between queueing a message and actually sending it",
```

Comment:
misspelled queuing

Reply:
Interesting.  For some twisted reason, this is the comment I chose to dig into the most.

* Oxford English Dictionary says "queueing", 
* Merriam-Webster says "queuing or queueing"
* Dictionary.com says "queuing"
* The Cambridge Dictionary says "queueing"
* Wikipedia says (at https://en.wikipedia.org/wiki/Queueing_theory):
> The spelling "queueing" over "queuing" is typically encountered in the academic research field. In fact, one of the flagship journals of the profession is Queueing Systems.

I think the snobs are on my side on this one. :)